studio/pages/project/[ref]/settings/billing/invoices.tsx

```diff
@@ -35,7 +35,7 @@ interface SettingsProps {
 
 const Settings: FC<SettingsProps> = ({ project }) => {
   const { ui } = useStore()
-  const projectTier = ui.selectedProject?.subscription_tier
+  const orgSlug = ui.selectedOrganization?.slug ?? ''
```

Comment:
Pretty sure there's a hook for org stuff now  — useOrganizationsQuery should have it?

Reply:
`useOrganizationsQuery` does an API call
`useParams` can't be used, as we only have the project ref in the path, not the org slug

Any other option?